Reqnroll.Generator/DefaultDependencyProvider.cs

```diff
@@ -45,6 +49,23 @@ public virtual void RegisterDefaults(ObjectContainer container)
 
             container.RegisterTypeAs<ConfigurationLoader, IConfigurationLoader>();
 
+            container.RegisterTypeAs<EnvironmentWrapper, IEnvironmentWrapper>();
+
+            // Section of registrations added for Cucumber Messages
+            container.RegisterTypeAs<FileService, IFileService>();
+            container.RegisterTypeAs<EnvVariableEnableFlagParser, IEnvVariableEnableFlagParser>();
+            container.RegisterTypeAs<FileBasedConfigurationResolver, ICucumberMessagesConfigurationResolver>("fileBasedResolver");
+            container.RegisterTypeAs<EnvironmentConfigurationResolver, ICucumberMessagesConfigurationResolver>("environmentBasedResolver");
+            container.RegisterFactoryAs<IEnumerable<ICucumberMessagesConfigurationResolver>>(() =>
```

Comment:
Might not be needed, see my comment on runtime dependency provider.

Reply:
acknowledged and comments above on impact.